src/main/kotlin/dev/ocpd/jsensible/rules/spring/SpringRules.kt

```diff
@@ -32,4 +35,19 @@ object SpringRules {
     fun noJavaxTransactionOnMethod() =
         noMethods().should().beAnnotatedWith("javax.transaction.Transactional")
             .because("in Spring application, it is recommended to use [org.springframework.transaction.annotation.Transactional] instead")
+
+    /**
+     * Only the [org.springframework.lang.Nullable] annotation is supported in Spring Data Repositories.
```

Comment:
`Spring Data Repositories only support their own [org.springframework.lang.Nullable] annoatation`

Reply:
It's annotation, not annoatation. 😸